main.js

```diff
@@ -160,4 +160,184 @@ const inventory = [
+  const tvElement = document.createElement('li');
+  tvElement.setAttribute("class","product");
+  const tvType = inventory[index].type;
+  tvElement.textContent = tvType;
+
+  const list = document.getElementById("tev1");
+  list.appendChild(tvElement);
+  console.log(tvElement, list);
+}
+for (let index = 0; index < inventory.lenght; index++) {
+  const product = inventory[index];
+  displayProduct(product);
+}
+displayProduct(inventory,0);
+displayProduct(inventory,1);
+
+
+//Opdracht 5a: Zorg ervoor dat er een string wordt gegenereerd voor de naam van een tv. Maak een functie die één tv-object als parameter verwacht en de naam op de volgende manier samenvoegt: [merk] [type] - [naam] zoals Philips 43PUS6504/12 - 4K TV of NIKKEI NH3216SMART - HD smart TV. Zorg ervoor dat je deze functie voor iedere tv zou kunnen gebruiken.
+function displayTelevisions(product) {
+  const tvElement = document.createElement('li');
+  tvElement.setAttribute('class', 'tv-list-item')
+  const tvType = `${product.brand}  ${product.type} - ${product.name}`;
+  tvElement.textContent = tvType;
+
+  const listofTV = document.getElementById('tvList')
+  listofTV.appendChild(tvElement);
+}
+for (let i = 0; i < inventory.length; i++) {
+  const product = inventory[i]
+  displayTelevisions(product)
+}
+
+///Opdracht 5b: Zorg ervoor dat de prijs van een tv netjes geformat wordt. Maak een functie die één tv-prijs als parameter verwacht (zoals 379) en daar de volgende string van maakt: €379,-. Zorg ervoor dat je deze functie voor iedere tv zou kunnen gebruiken.
+function priceOke(pricing) {
+  const tvpElement = document.createElement('p');
+  tvpElement.setAttribute('class', 'prijs-item');
+  const tvPrice = `${pricing.name}, € ${pricing.price},- `;
+  console.log(tvpElement);
+  tvpElement.textContent = tvPrice;
+
+  const listOfPrice = document.getElementById('prijs');
+  listOfPrice.appendChild(tvpElement);
+  //console.log(tvElement, listsOfPrice);
+}
+for (let i = 0; i < inventory.length; i++)  {
+  const pricing = inventory[i];
+  priceOke(pricing);
+}
+
+
+//Opdracht 5c: Zorg ervoor dat er een string wordt gegenereerd voor alle beschikbare schermgroottes van één tv in zowel inches als cm Maak een functie die één screen-sizes array verwacht en de groottes op de volgende manier samenvoegt: [schermgrootte] inches ([schermgrootte omgerekend]cm) | [schermgrootte] inches ([schermgrootte omgerekend]cm) etc. Dus een input van [32] geeft 32 inch (81 cm) en een input van [43, 50, 55, 58] geeft 43 inch (109 cm) | 50 inch (127 cm) | 58 inch (147 cm). Zorg ervoor dat je deze functie voor iedere tv zou kunnen gebruiken, zowel voor tv's met maar één schermgrootte als met tientallen schermgroottes.
+function size(groote, index) {
+  const tvSize = document.createElement('p');
+  tvSize.setAttribute('class', 'afmeting');
+  //for (const groote of groot) {
+    const tvGrote = `${groote.availableSizes} inch ${groote.availableSizes * 2.54} cm`;
```

Comment:
`availableSizes` is een array en geen numeric value. Je had moeten itereren over deze array heen, zodat je dan voor elke size dit had kunnen doen